src/main.rs

```diff
@@ -199,6 +199,8 @@ fn main() {
     if let Ok(filepath) = dotenvy::var("PF_SOURCE") {
         dotenvy::from_path(filepath).unwrap();
     }
+    // Check if SKIP_SLOW is enabled
+    let skip_slow = dotenvy::var("SKIP_SLOW").map_or(false, |val| val == "true");
```

Comment:
1. Rename `skip_slow` to `skip_slow_package_managers` here as well
2. Rename `SKIP_SLOW` to `PF_FAST_PKG_COUNT` to make the variable name pfetch specific
3. use `.is_ok()` instead of `.map_or(false, |val| val == "true")` to allow the env variable to be set to any value